internal/resources/metal/metal_project_ssh_key/framework_datasource.go

```diff
@@ -0,0 +1,99 @@
+package metal_project_ssh_key
```

Comment:
I think the directory and package name could be simplified to not repeat the parent directory, project_ssh_key, in this case.

Similarly, I don't see added value in the `framework_` prefixes. It may make it easier to find framework vs sdkv2 files, but the contents, types and methods, of those files are just as telling.

Reply:
addressed in https://github.com/equinix/terraform-provider-equinix/pull/406/commits/b8cc18d3e9e193006d8a6cbadbebd7f28a565bad